services/api-server/.env-devel

```diff
@@ -45,3 +45,6 @@ STORAGE_HOST=storage
 
 # director
 DIRECTOR_V2_HOST=director-v2
+
+# openapi generator version
+OPENAPI_GENERATOR_VERSION=v4.2.3
```

Comment:
I did it like this because of the make fcn which fetches the templates. I want to make sure the templates and the generator have the same version. So I guess what I should do is add this directly in the `Makefile` in `services/api-server` and then leave the tool expecting the version defined as an environment variable. OK with you?